hugegraph-api/pom.xml

```diff
@@ -186,7 +186,7 @@
                         </manifest>
                         <manifestEntries>
                             <!-- TODO: update it -->
-                            <Implementation-Version>0.69.0.0</Implementation-Version>
+                            <Implementation-Version>${project.version}</Implementation-Version>
```

Comment:
seems we should keep it now, it's APIversion

Reply:
@javeme  if we should keep all version to 1.0.0 ?

Reply:
I think we need to keep the api history